homeassistant/components/homeassistant/scene.py

```diff
@@ -271,6 +284,10 @@ async def create_service(call: ServiceCall) -> None:
         SCENE_DOMAIN, SERVICE_CREATE, create_service, CREATE_SCENE_SCHEMA
     )
 
+    platform.async_register_entity_service(
```

Comment:
Let's move back to using `hass.services.async_register` but create the validation schema using the `cv.make_entity_service_schema` helper. We can do that inline I think.

Reply:
We can use `async_extract_entity_ids` in the service helper to get the `entity_id`s from the device_ids and area_ids:
https://github.com/home-assistant/core/blob/9ed745638df5e2ae44d5d405a897690518fb6950/homeassistant/helpers/service.py#L433-L440

Reply:
But do we want that? Who assigns dynamically created scenes to areas? And do devices even exist in this context?

The service just deletes a small subset of scenes and takes an `entity_id` as an input. What good would this do?

Reply:
Yeah, I guess it's an edge case. Ok, let's not use `cv.make_entity_service_schema` for now. If we see a need later we can always add it later.

Reply:
Too late, I used it and pushed the commit. I guess it won't hurt, although it's certainly overkill.

Reply:
I just want to get this (and [this](https://github.com/home-assistant/core/pull/91165)) merged to fix a problem I've been having for almost a year, so if you want the helper, sure, let's engineer the crap of out it :)